sklearn_numba_dpex/kmeans/kernels/compute_euclidean_distances.py

```diff
@@ -82,32 +84,35 @@ def compute_distances(
 
             for _1 in range(n_windows_for_features):
                 is_last_feature_window = _1 == last_feature_window_idx
+
                 load_window_of_centroids_and_features(
                     first_feature_idx,
                     loading_centroid_idx,
                     window_loading_centroid_idx,
                     window_loading_feature_offset,
                     current_centroids_t,
                     centroids_window,
-                    is_last_feature_window
                 )
 
                 dpex.barrier(dpex.CLK_LOCAL_MEM_FENCE)
-                accumulate_sq_distances(sample_idx, first_feature_idx, X_t, centroids_window, sq_distances, is_last_feature_window, is_last_centroid_window)
 
-                dpex.barrier(dpex.CLK_LOCAL_MEM_FENCE)
+                accumulate_sq_distances(
+                    sample_idx, first_feature_idx, X_t, centroids_window, sq_distances, is_last_feature_window, is_last_centroid_window
+                )
```

Comment:
Can this be `blackified`?

Reply:
This is blackified already. black does not always try to fit the max line length requirement. If you want to hard enforce this another PR should be opened to configure it properly imo.